src/compiler/CompileContext.js

```diff
@@ -330,6 +330,17 @@ class CompileContext extends EventEmitter {
         return this._errors;
     }
 
+    getRelativePath(importPath) {
```

Comment:
`getRequirePath` should handle both relative paths and modules.  Do we need this new function?

Reply:
It does, however what happens is it tries to resolve the module and we have tests that are importing files that do not exist which fail.
I spoke to @DylanPiercey about this, he told me to add this function since the other tests were failing.